third_party/proxy/config/config.go

```diff
@@ -330,7 +330,7 @@ func (c *EndpointSliceConfig) handleAddEndpointSlice(obj interface{}) {
 		return
 	}
 	for _, h := range c.eventHandlers {
-		klog.V(4).Infof("Calling handler.OnEndpointSliceAdd %+v", endpointSlice)
+		klog.V(4).InfoS("Calling handler.OnEndpointSliceAdd", endpointSlice)
```

Comment:
structured logging must have pairs of key and value, otherwise the log wouldn't be formated properly.
```
InfoS("log body", "key1", value1, "key2", values)
```
It should be `klog.V(4).InfoS("Calling handler.OnEndpointSliceAdd", "endpointSlice", endpointSlice)`

Reply:
Updated, thanks @tnqn.